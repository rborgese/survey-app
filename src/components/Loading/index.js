import * as React from 'react';

const Loading = () => {
  return (
    <div className="loading">
      <h1 className="text-center" data-testid="loading-text">
        Loading...
      </h1>
    </div>
  );
};

export default Loading;
