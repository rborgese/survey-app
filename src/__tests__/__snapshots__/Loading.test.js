import React from 'react';
import renderer from 'react-test-renderer'
import Loading from '../../components/Loading'

// check the loading test snapshot
test('Loading renders correctly', () => {
    const component = renderer.create(<Loading/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})

