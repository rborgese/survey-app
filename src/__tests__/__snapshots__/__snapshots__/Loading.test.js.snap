// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<div
  className="loading"
>
  <h1
    className="text-center"
    data-testid="loading-text"
  >
    Loading...
  </h1>
</div>
`;
